### PR TITLE
Troubleshooting - namespace stuck terminating

### PR DESCRIPTION
**Pull request by domgreen**
**What type of PR is this?**
/kind documentation

**What this PR does / Why we need it**:

Adding documentation around what to do if you accidentally get your namespace stuck in terminating.

**Which issue(s) this PR fixes**:

Links #1778 
Links #1783

### CODE REVIEW

File: site/content/en/docs/Guides/troubleshooting.md
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:368]: Possibly we need to clear this a bit:
```suggestion
You may need to clean up any other Agones related resources you have in your cluster at this point.
```

File: site/content/en/docs/Guides/troubleshooting.md
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:353]: This is not a valid JSON excerpt. But I see that this was taken from an article.
```suggestion
         "finalizers": [
             "kubernetes"
         ]
```

**Review by aLekSer — APPROVED** [review]
I verified the instructions you added, they actually was helpful. Please fix the JSON. I get an error when I am trying to use `"finalizers: "` with an empty value: 
```
curl -k -H "Content-Type: application/json" -X PUT --data-binary @tmp.json http://127.0.0.1:8001/api/v1/namespaces/agones-system/finalize
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {

  },
  "status": "Failure",
  "message": "the object provided is unrecognized (must be of type Namespace): couldn't get version/kind; json parse error: invalid character '}' looking for beginning of value ({\n    \"apiVersion\": \"v1\",\n     ...)",
  "reason": "BadRequest",
  "code": 400
```

**Comment by markmandel** [comment]
@aLekSer just wanted to check - I was about to merge this, but I'm not sure if you mean for changes to be made before getting merged?

**Comment by domgreen** [comment]
> @aLekSer just wanted to check - I was about to merge this, but I'm not sure if you mean for changes to be made before getting merged? 

Think I need to fix up the JSON ... Will look this morning 👍

**Comment by aLekSer** [comment]
Yes, overall look great, but a small nit is a not valid JSON without a value, which looks not perfect when someone read the docs.

**Comment by domgreen** [comment]
Thanks @aLekSer missed that it was a GH recommendation otherwise I would have clicked it last night :/ 

@markmandel once were green I think were good to merge :+1:

**Pull request merged**
